packages/trimerge-sync/src/CoordinatingLocalStore.ts

```diff
@@ -254,7 +254,9 @@ export class CoordinatingLocalStore<CommitMetadata, Delta, Presence>
         if (origin === 'remote' && event.fatal) {
           // Do not await on this or we'll deadlock
           void this.closeRemote(event.reconnect !== false);
-          if (event.code === 'unauthorized') {
+          // If the remote indicated that reconnecting won't address the issues
+          // we forward the error event to trimerge client.
+          if (!event.reconnect) {
```

Comment:
We previously wanted to avoid showing an error screen on 401 errors so we were explicitly forwarding 'unauthorized' errors. Now, since PullPokeRemote handles 'unauthorized' errors as non reconnectable the behavior should be the same and if for whatever reason we did want to set `reconnect: true` we wouldn't want to emit that error anyway, since it would mean that we'd attempt to retry the request.